.ci/travis.sh

```diff
@@ -37,6 +37,10 @@ run-command)
   fi
   ;;
 
+quarterly-cache-cleanup)
+  find ~/.m2 -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
```

Comment:
`maxdepth` is to not go too far into tycho. We can exclude a specific directory, but it doesn't seem possible to exclude it's children. So this was needed for that.

`-type d` is to only target full directory deletions.